### ISSUE TITLE
loss계산이 잘 안되네요....

### ISSUE THREAD

**Issue by Changsik-K** [cosmoquester/speech-recognition#37]
프로그램을 실행하면 아래처럼 loss와 accuracy 계산이 안됩니다. 
warning 부분도 의심스럽습니다. 

[2022-04-15 13:03:36,067] [+] Model compile
WARNING:tensorflow:tf.keras.mixed_precision.experimental.LossScaleOptimizer is deprecated. Please use tf.keras.mixed_precision.LossScaleOptimizer instead. Note that the non-experimental LossScaleOptimizer does not take a DynamicLossScale but instead takes the dynamic configuration directly in the constructor. For example:
  opt = tf.keras.mixed_precision.LossScaleOptimizer(opt)

[2022-04-15 13:03:36,146] [+] Repeat dataset
[2022-04-15 13:03:36,149] [+] Pad Input data
[2022-04-15 13:03:36,378] [+] Start training
Epoch 1/100
40/40 [==============================] - ETA: 0s - loss: nan - accuracy: 0.0000e+00 
Epoch 1: saving model to output/models/model-1epoch-nanloss_0.0000acc.ckpt
40/40 [==============================] - 955s 21s/step - loss: nan - accuracy: 0.0000e+00 - val_loss: nan - val_accuracy: 0.0000e+00
Epoch 2/100
40/40 [==============================] - ETA: 0s - loss: nan - accuracy: 0.0000e+00 
Epoch 2: saving model to output/models/model-2epoch-nanloss_0.0000acc.ckpt
40/40 [==============================] - 840s 21s/step - loss: nan - accuracy: 0.0000e+00 - val_loss: nan - val_accuracy: 0.0000e+00
Epoch 3/100
40/40 [==============================] - ETA: 0s - loss: nan - accuracy: 0.0000e+00 
Epoch 3: saving model to output/models/model-3epoch-nanloss_0.0000acc.ckpt
40/40 [==============================] - 866s 21s/step - loss: nan - accuracy: 0.0000e+00 - val_loss: nan - val_accuracy: 0.0000e+00
Epoch 4/100
29/40 [====================>.........] - ETA: 3:42 - loss: nan - accuracy: 0.0000e+00
